README.md

```diff
@@ -230,6 +230,11 @@ The custom configuration options for the Celery workers are listed below:
   Additionally, it will use this file as a base and set the `overwrite_from_index_token` for the
   registry of the `from_index` container image when applicable. IIB will never directly modify this
   file though. This defaults to `~/.docker/config.json.template`.
+*  `iib_dogpile_backend` - configuration for dogpile.cache backend. Default value is
+   `'dogpile.cache.null'`. In case you want to enable caching use `'dogpile.cache.memcached'`.
```

Comment:
```suggestion
*  `iib_dogpile_backend` - The configuration for the dogpile.cache backend. The default value is
   `'dogpile.cache.null'`. In case you want to enable caching, set this to `'dogpile.cache.memcached'`.
```